### PR TITLE
Completed assignment

### PR DESCRIPTION
**Pull request by BharatKatyal**

### CODE REVIEW

**Review by BharatKatyal — COMMENTED** [review]
reuploaded

File: __workshop/exercise-1/1-questions.md [hunk 1]
```diff
@@ -1,63 +1,84 @@
...
+b) [F]
 
 ```html
 <ul>
 <li>one</li>
 </ol>
 ```
 
-c) [ ] `<ul></ul><img/><ol><li>one</li></ol>`
+c) [F] `<ul></ul><img/><ol><li>one</li></ol>`
```
Comment (orangesmartie92) [inline __workshop/exercise-1/1-questions.md:18]: true.  tags are ordered properly and are closed properly.

File: __workshop/exercise-1/1-questions.md [hunk 2]
```diff
@@ -1,63 +1,84 @@
...
 c) You want to sell designer hats. You need to receive orders from the user.
 
+
 ## Q4 - Can a `button` be a child of a `button`? Explain your reasoning
 
-## Q5 - What is the most generic tag you can use?
+you cannot, a button is not ment to contain another element 
 
+## Q5 - What is the most generic tag you can use?
+html head body and title
```
Comment (orangesmartie92) [inline __workshop/exercise-1/1-questions.md:50]: div is the most generic as it only represents a block element.

File: __workshop/exercise-1/1-questions.md [hunk 3]
```diff
@@ -1,63 +1,84 @@
...
 a) You want to create a webpage with the photos from your latest vacation
+<IMG> tag, the source of image.
+
+
+<img src=”/”>
+
+
 
 b) You want to create a website that lists all the art gallery websites in your city and links to their website.
+It depends how you want the website setup. I would say h1- for the main title, h2's stating
```
Comment (orangesmartie92) [inline __workshop/exercise-1/1-questions.md:40]: you'd need ul, li, and a tags for a list of links.

File: __workshop/exercise-1/1-questions.md [hunk 4]
```diff
@@ -1,63 +1,84 @@
...
+<img src=”/”>
+
+
 
 b) You want to create a website that lists all the art gallery websites in your city and links to their website.
+It depends how you want the website setup. I would say h1- for the main title, h2's stating 
 
 c) You want to sell designer hats. You need to receive orders from the user.
 
+
```
Comment (orangesmartie92) [inline __workshop/exercise-1/1-questions.md:44]: missing answer

File: __workshop/exercise-1/1-questions.md [hunk 5]
```diff
@@ -1,63 +1,84 @@
...
 
 ## Q8 - What is the difference between td and th?
+th is ment to label the header and td is data inside the cells underneath the header 
 
 ## Q9 - Which tag makes the text appear bigger: h1 or h3?
+h1 because its the main heading 
 
 ## Q10 - In which situation can you use self closing tags?
-
+in modern html self closing tags are not allowed, they were used in situations in such where there is no context between them such as <p></p> (with no space at all in between), you’d be able write <p/>.
```
Comment (orangesmartie92) [inline __workshop/exercise-1/1-questions.md:77]: self closing tags can be used with the following : refer to http://xahlee.info/js/html5_non-closing_tag.html for a full list

File: __workshop/exercise-1/1-questions.md [hunk 6]
```diff
@@ -1,63 +1,84 @@
...
 ## Q11 - What is autofilling and why is it important?
+Autocomplete helps the user complete teh task quicker by providing recomendations
```
Comment (orangesmartie92) [inline __workshop/exercise-1/1-questions.md:79]: autocomplete attribute helps the browser determine which inputs to autocomplete.

File: __workshop/exercise-7/index.html [hunk 7]
```diff
@@ -1 +1,49 @@
 <!-- ...crickets... -->
```
Comment (orangesmartie92) [inline __workshop/exercise-7/index.html:1]: Missing
```
<!DOCTYPE html>
<html>
  <body>
(...)
```

File: __workshop/exercise-8/index.html [hunk 8]
```diff
@@ -1 +1,42 @@
...
+<input type="checkbox" id="extra1" name="extra1" value="Baby_seat">
+<label for="extra1">Baby Seat</label>
+<input type="checkbox" id="extra2" name="extra2" value="wheelchair">
+<label for="extra2">wheelchair access</label>
+<input type="checkbox" id="extra3" name="extra3" value="privacy">
+<label for="extra3">Privacy windows</label><br>
+
+<h2>Choose Your Pickup location</h2>
+<label for="cars">Pickup Location:</label>
+  <select name="cars" id="cars">
```
Comment (orangesmartie92) [inline __workshop/exercise-8/index.html:30]:
```
<option disabled selected>Select One</option>
```
to signify that you must select one

File: __workshop/exercise-8/index.html [hunk 9]
```diff
@@ -1 +1,42 @@
 <!-- ...crickets... -->
```
Comment (orangesmartie92) [inline __workshop/exercise-8/index.html:1]: missing
```
<!DOCTYPE html>
<html>
  <body>
(...)
```

**Review by orangesmartie92 — APPROVED** [review]

**Pull request closed without merging**